README.md

```diff
@@ -2,27 +2,47 @@
 Tracker to check the covid vaccine slot availability in your district and send mobile notifications through Telegram.
 
 ## Requirements
-Docker must be installed in the local system. Refer [docker documentation](https://docs.docker.com/engine/install/) to set it locally based on your machine specification.
-If you are a Windows user and wants to setup docker, follow this [video](
-https://youtu.be/_9AWYlt86B8)
+Telegram App account
+Anaconda
 
 ## How to use?
 There are two parts to this system
 1. Pinging the public COWIN API to get district wise data and checking for availability, every 15 minutes.
-2. Relaying this information to the user's mobile via Telegram Bot. For that, you need to create a bot. Follow this [tutorial](https://sendpulse.com/knowledge-base/chatbot/create-telegram-chatbot) to create a telegram bot. Refer the `How to Create a New Bot for Telegram` section. Note the `TOKEN`.
-3. Now, open Telegram App and type `@Get_Channel_User_Telegram_ID_Bot`, and go to the chat. Type something and this should help you get your `Telegram ID`, which is your chat id. Note that as well.
-4. After successfully completing the above steps, you are good to go.
+2. Open Telegram App and type `@Get_Channel_User_Telegram_ID_Bot` or go to https://t.me/Get_Channel_User_Telegram_ID_Bot and type Hi in the chat
+You should get something like
+![TeleGet](telegram_get_id.PNG)
+
+Use your Telegram id and add it to setting.json file in Telegram_ID section
+Eg:
+{
+  "districtId": "1",
+  "Telegram_ID": "18353498",
+  "userAge": "45"
+}
+
+3. Now, open the link http://t.me/CowinSlotTrackerBot and click on Start.
+4. After successfully completing the above steps, install Anaconda using the link
+https://docs.anaconda.com/anaconda/install/ depending on your operating system i.e Windows, Linux, MAC
```

Comment:
use [Anaconda](https://docs.anaconda.com/anaconda/install/) instead of just the link @trishh088